crates/bevy_text/src/bounds.rs

```diff
@@ -0,0 +1,70 @@
+use bevy_ecs::{component::Component, reflect::ReflectComponent};
+use bevy_math::Vec2;
+use bevy_reflect::Reflect;
+
+/// The maximum width and height of text. The text will wrap according to the specified size.
+/// Characters out of the bounds after wrapping will be truncated. Text is aligned according to the
+/// specified [`JustifyText`](crate::text::JustifyText).
+///
+/// Note: only characters that are completely out of the bounds will be truncated, so this is not a
```

Comment:
I know this comment is from the old `Text2dBounds` struct, but is this behavior the same in cosmic text?

Reply:
Non-blocking